flake.nix

```diff
@@ -69,7 +69,7 @@
             $out/bin/jj support completion --bash > ./completions.bash
             installShellCompletion --bash --name ${pname}.bash ./completions.bash
             $out/bin/jj support completion --fish > ./completions.fish
-            installShellCompletion --fish --name ${pname}.fish ./completions.fish
+            installShellCompletion --fish --name jj.fish ./completions.fish
```

Comment:
Why would this a problem for this line only and not for the other lines using `${pname}`?

Reply:
I've actually decided to test other shells, and while `zsh` worked, `bash` required similar change, you were right.

In the end, I've looked at how nixpkgs packages do `installShellCompletion` and found a cleaner way.

Reply:
I prefer this way indeed.